gamestonk_terminal/options/chains_view.py

```diff
@@ -124,25 +136,26 @@ def display_chains(symbol: str, expiry: str, other_args: List[str]):
         default=-1,
         help="maximum strike price to consider.",
     )
+    parser.add_argument(
+        "-d",
+        "--display",
+        dest="to_display",
+        default=default_columns,
+        help="columns to look at",
+    )
 
     chains_df = get_option_chains(symbol, expiry)
 
-    default_columns = [
-        "mid_iv",
-        "delta",
-        "volume",
-        "open_interest",
-        "bid",
-        "ask",
-        "strike",
-        "option_type",
-    ]
-    chains_df = chains_df[default_columns].rename(columns=column_map)
-
     try:
         ns_parser = parse_known_args_and_warn(parser, other_args)
         if not ns_parser:
             return
+        if isinstance(ns_parser.to_display, str):
+            columns = list(ns_parser.to_display.split(",")) + ["strike", "option_type"]
+        if isinstance(ns_parser.to_display, list):
+            columns = ns_parser.to_display + ["strike", "option_type"]
```

Comment:
This wouldn't be needed, as the ns_parser.to_display would contain the `List[str]` regardless